include/cantera/kinetics/ReactionRateFactory.h

```diff
@@ -19,15 +19,39 @@ namespace Cantera
 class Kinetics;
 class Units;
 
+/**
+ * @defgroup arrheniusGroup  Arrhenius-type Parameterizations
+ * Classes implementing the standard Arrhenius rate parameterization and derived models.
+ * @ingroup reactionGroup
+ */
+
+/**
+ * @defgroup falloffGroup  Falloff Parameterizations
+ * Classes implementing fall-off in reaction rate constants due to intermolecular energy
+ * transfer and derived models.
+ * @ingroup reactionGroup
+ */
+
+/**
+ * @defgroup surfaceGroup  Coverage-Dependent Rate Parameterizations
+ * Classes implementing reaction rates that involve interfaces.
+ * @ingroup reactionGroup
```

Comment:
I think "Interface Rate Parameterizations" is a better description for this group -- sticking rates don't have any dependence on the coverages.